documentation/staging/content/faq/security-validation.md

```diff
@@ -0,0 +1,25 @@
+---
+title: "Handling security validations"
+date: 2020-06-30T08:55:00-05:00
+draft: false
+weight: 14
+description: "Why am I seeing these security warnings?"
+---
+
+> After applying the July2021 PSU, I'm now seeing security warnings, such as:
+>
+> Description: Production Mode is enabled but user lockout settings are not secure in realm: myrealm, i.e. LockoutThreshold should not be greater than 5, LockoutDuration should not be less than 30.
+>
+> SOLUTION: Update the user lockout settings (LockoutThreshold, LockoutDuration) to be secure.
+
+WebLogic Server has a new, important feature to ensure and help you secure your WLS domains when running in production. With the July 2021 PSU applied, WebLogic Server regularly validates your domain configuration settings against a set of security configuration guidelines to determine whether the domain meets key security guidelines recommended by Oracle. For more information and additional details, see [MOS Doc 2788605.1](https://support.oracle.com/rs?type=doc&id=2788605.1) "WebLogic Server Security Warnings Displayed Through the Admin Console" and [Review Potential Security Issues](https://docs.oracle.com/en/middleware/fusion-middleware/weblogic-server/12.2.1.4/lockd/secure.html#GUID-4148D1BE-2D54-4DA5-8E94-A35D48DCEF1D) in _Securing a Production Environment for Oracle WebLogic Server_.
+
+Warnings may be at the level of the JDK, or that SSL is not enabled. They may recommend that you take an external action like setting the right permissions in the `DOMAIN_HOME` file system. Some warnings may recommend updating your WebLogic configuration.
```

Comment:
I have opened https://jira.oraclecorp.com/jira/browse/OWLS-91448 addressing file permissions of the Domain Home.  We have found that in OpenShift when WDT creates the the Domain Home the Operator needs write permissions.  Not sure if this is only a OpenShift issue or also in other K8s vendors.

Reply:
I would remove for now the sentence "They may recommend that you take an external action like setting the right permissions in the `DOMAIN_HOME` file system." until we haver more clarity

Reply:
Done. See https://github.com/oracle/weblogic-kubernetes-operator/pull/2495/commits/821a6e7bc2f1ded427f61db523bd39951d7c03d5